src/main/java/com/aws/greengrass/builtin/services/configstore/ConfigStoreIPCEventStreamAgent.java

```diff
@@ -402,10 +404,14 @@ public SubscribeToConfigurationUpdateResponse handleRequest(SubscribeToConfigura
             });
         }
 
+        @Override
+        public void afterHandleRequest() {
+            active.set(true);
```

Comment:
should it be set to false in on `onStreamClosed` or is that a scenario that doesn't matter?

Reply:
Shouldn't matter. When the stream is closed, the _send_ method won't do anything anyway.